src/lib/components/Sidebar.svelte

```diff
@@ -22,6 +22,12 @@ The sidebar/drawer.
         document.activeElement.blur();
     };
     beforeNavigate(closeDrawer);
+
+    let showSearch = $globalConfig.mainFeatures['search'];
+    let showHistory = $globalConfig.mainFeatures['history'];
+    let showBookmarks = $globalConfig.mainFeatures['annotation-bookmarks'];
+    let showNotes = $globalConfig.mainFeatures['annotation-notes'];
+    let showHighlights = $globalConfig.mainFeatures['annotation-highlights'];
```

Comment:
Are annotation-bookmarks, annotation-notes, and annotation-highlights exclusively settings for the annotation menu or should they be modifying the drawer too?

Reply:
@jakedcolbert You can build an Android app to check.

Reply:
They modify the drawer as well.